css/custom.css

```diff
@@ -4218,8 +4218,16 @@ p.hangouts-social {
     border: none;
 }
 
+.sponsors p {
+    padding-top: 2rem;
+}
+
+.sponsors p#tari {
+    padding-top: 3rem;
+}
+
 .sponsors img {
-    width: 100%;
+    max-height: 100px;
 }
 
```

Comment:
```suggestion
.sponsors .info-block {
    display: flex;
    justify-content: center;
    align-items: center;
}
```

Reply:
Thanks, that was what I was looking for :)

Reply:
Thanks @l33d4n. Added.